### DISCUSSION TITLE
a new OS for x86?

### DISCUSSION THREAD

**Discussion by josef2600** [wwarthen/RomWBW#412] [General]
@ Wayne Warthen .
hi, 
i was just rooming around git, and was searching about some overflowing stuff, and in something not really relevant, i found reply posts from  chuck todd .then trough him i found your git. believe me, i never thought i would fined any one at this level ! usually either they are dead or very very old and long out of the business. the way chuck todd explains hardware work, amazes me! i have not seen it in at least 16 years. and then i met your code! f me! i thought i was the last one standing in old fashion electronics. now days, nobody even understand what a CPU is. they cant write a single good code. it is so hard nobody cares or understand why the hell i care about those things. 
this is me literally bowing to you both. thank you for simply caring and working. lets just say, in the 1970's , humen sent 24 people to the moon and 12 of them walked on it and their hole ships only had a 4KB of f-ing DRAM!!! now i open 2 pages of YouTube and system goes mad with 64 GB of ram! 
enough ranting. 
the thing that got me thinking, was the replacement OS for windows. no, don't care about Linux, it is not good enough. simply put, it doesn't have any engineering software. i don't mind closed source of the windows, i actually like it. and i don't care about its price too. it is fine. i care about that they don't care about you and me. they don't want you to have any control over what you own, simply, you don't own anything on this earth! even my thoughts are not my own, they own it. like apple.
but a project like react os [](https://reactos.org/) is a good place to place your interest on. no, i am not related to them in any way, i am not even using that thing! why? because they are not real coders! but if you and some other gods like you can help them, i am sure it will be perfect. 
in my opinion, it doesn't matter if it supports 486 or not, it is just a waste of talent and time. instead just work with what the newest version of windows work with. one of the biggest challenges will be with Direct X implementation. from 9 to 12. it can not have it in it, but anyone should be able to install it on it. for copy write issues! and the same for dot net. and for default the name version can be what ever (like react v xx), but user can change it to something like win 10 xxxx or win 11 version xxxx. because software's are now lucked for a specific version of windows, because they can steal everyone's data. and many many things that i am not sure i can say here, some body might get offended for some shhh!
it would be perfect if you and some of other people in your league can work on something like this.
thank you again for being. thank you.
best regards, Josef.

**Comment by wwarthen** [comment]
Hi @josef2600,

I sincerely appreciate your kind words.  Indeed, RomWBW is a labor of love.

Sadly, I simply don't have time to pursue an additional ambitious project such as you suggest.  I hope there are others that will take up the gauntlet.

Thanks, Wayne
